drivers/firmware/nrf_ironside/cpuconf.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include <zephyr/sys/util.h>
+#include <zephyr/kernel.h>
+
+#include <nrfx.h>
```

Comment:
This is redundant here, as `zephyr/drivers/firmware/nrf_ironside/cpuconf.h` includes it.

Reply:
fixed in https://github.com/zephyrproject-rtos/zephyr/pull/89333